docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -0,0 +1,266 @@
+
+```diff
+<DatePicker
+-  renderDay={(_, dayProps) => <CustomDay {...dayProps} />}
++  components={{ Day: CustomDay }}
+/>
+```
+
+### Rename the localization props
+
+The props used to set the text displayed in the pickers have been replaced by keys inside the `localeText` prop:
+
+| Removed prop                 | Property in the new `localText` prop                                              |
+| ---------------------------- | --------------------------------------------------------------------------------- |
+| `endText`                    | `end`                                                                             |
+| `getClockLabelText`          | `clockLabelText`                                                                  |
+| `getHoursClockNumberText`    | `hoursClockNumberText`                                                            |
+| `getMinutesClockNumberText`  | `minutesClockNumberText`                                                          |
+| `getSecondsClockNumberText`  | `secondsClockNumberText`                                                          |
+| `getViewSwitchingButtonText` | `calendarViewSwitchingButtonAriaLabel`                                            |
+| `leftArrowButtonText`        | `openPreviousView` (or `previousMonth` when the button changes the visible month) |
+| `rightArrowButtonText`       | `openNextView` (or `nextMonth` when the button changes the visible month)         |
+| `startText`                  | `start`                                                                           |
+
+For instance if you want to replace the `startText` / `endText`
+
+```diff
+<DateRangePicker
+-  startText="From"
+-  endText="To"
++  localeText={{
++    start: 'From',
++    end: 'To',
++  }}
+/>
+```
+
+### Rename the `locale` prop on `LocalizationProvider`
+
+The `locale` prop of the `LocalizationProvider` component have been renamed `adapterLcoale`:
+
+```diff
+<LocalizationProvider
+  dateAdapter={AdapterDayjs}
+-  locale="fr"
++  adapterLocale="fr"
+>
+  {children}
+</LocalizationProvider
+```
+
+### Rename the view components
+
+The view components allowing to pick a date or parts of a date without an input have been renamed to better fit their usage:
+
+```diff
+-<CalendarPicker {...props} />
++<DateCalendar  {...props} />
+```
+
+```diff
+-<DayPicker {...props} />
++<DayCalendar  {...props} />
+```
+
+```diff
+-<CalendarPickerSkeleton {...props} />
++<DayCalendarSkeleton  {...props} />
+```
+
+```diff
+-<MonthPicker {...props} />
++<MonthCalendar  {...props} />
+```
+
+```diff
+-<YearPicker {...props} />
++<YearCalendar  {...props} />
+```
+
+Component names in the theme have changed as well:
+
+```diff
+-MuiCalendarPicker: {
++MuiDateCalendar: {
+```
+
+```diff
+-MuiDayPicker: {
++MuiDayCalendar: {
+```
+
+```diff
+-MuiCalendarPickerSkeleton: {
++MuiDayCalendarSkeleton: {
+```
+
+```diff
+-MuiMonthPicker: {
++MuiMonthCalendar: {
+```
+
+```diff
```

Comment:
```suggestion

```

Could put all the examples in the same diff to be consistent with the last section

![image](https://user-images.githubusercontent.com/45398769/196369551-913ee4e6-b6e1-4a7a-a726-18a61a2ad160.png)